Vostok.Logging.Microsoft.Tests/Vostok.Logging.Microsoft.Tests.csproj

```diff
@@ -1,32 +1,41 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  
-  <Import Project="..\..\vostok.devtools\library-common-props\Test-Project.props" />
-  <Import Project="..\..\vostok.devtools\git-commit-to-assembly-title\Vostok.Tools.GitCommit2AssemblyTitle.props" />
-  
+
+  <Import Project="..\..\vostok.devtools\library-common-props\Test-Project.props"/>
+  <Import Project="..\..\vostok.devtools\git-commit-to-assembly-title\Vostok.Tools.GitCommit2AssemblyTitle.props"/>
+
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.1;net471</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1</TargetFrameworks>
     <LangVersion>7.2</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="5.4.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
-    <PackageReference Include="NSubstitute" Version="3.1.0" />
-    <PackageReference Include="NUnit" Version="3.10.1" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
+    <PackageReference Include="FluentAssertions" Version="5.4.1"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0"/>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3"/>
+    <PackageReference Include="NSubstitute" Version="3.1.0"/>
+    <PackageReference Include="NUnit" Version="3.10.1"/>
+    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0"/>
+
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="2.2.0"/>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0"/>
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.2.0"/>
+    <PackageReference Include="Vostok.Commons.Collections" Version="0.1.3"/>
```

Comment:
Please reference `vostok.commons.collections` module from Cement to preserve unified approach to internal dependencies.